src/App.php

```diff
@@ -1502,4 +1505,26 @@ public function redirect($toUrl)
 			$this->internalRedirect($toUrl);
 		}
 	}
+
+	/**
+	 * @brief Static declaration of Friendica routes.
```

Comment:
I thought `@brief` isn't "really" supported. I had such change requested some time ago :-)

Reply:
You're right, I didn't provide a comprehensive comment without the `@brief` . The problem is that it is non-standard, so the comment must make sense even without it.